components/email-input.jsx

```diff
@@ -58,10 +71,7 @@ var EmailInput = React.createClass({
             <div className="field-container">
               <i className="fa fa-envelope field-icon"></i>
               <input type="email" className={inputClassName} name="email" value={this.state.values.email} onChange={this.onEmailChange} placeholder={this.getIntlMessage('email')}/>
-              <i onClick={this.hintClicked} className={hintClassIconName}></i>
-              <div className={hintClassName}>
-                <FormattedHTMLMessage message={ this.getIntlMessage("email_info") } />
-              </div>
+              {this.renderHint()}
```

Comment:
This is a new pattern I am trying for adding conditions in react markup. I kinda like it more than just making a whole new component because the `this` param is now the same in the conditional render, and no longer needs things passed to it.